README.md

```diff
@@ -100,29 +99,26 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Create a relational database]**
+- **[Use SQL to querry data from database tables]**
+- **[Update and insert data using SQL queries]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Live Demo Link](<>)
 
```

Comment:
- [ ] It would be nice if you remove this Live Demo section from your readme file to make it clean and descriptive :+1: Once you will have your live link then you can add this section again with the live link :blush: